lib/membrane_element_aac/encoder.ex

```diff
@@ -0,0 +1,277 @@
+defmodule Membrane.Element.AAC.Encoder do
+  @moduledoc """
+  Element encoding raw audio into AAC format
+  """
+
+  use Membrane.Element.Base.Filter
+  use Bunch.Typespec
+  alias __MODULE__.Native
+  alias Membrane.Buffer
+  alias Membrane.Caps.Audio.Raw
+  alias Membrane.Caps.Matcher
+  alias Membrane.Event.EndOfStream
+
+  use Membrane.Log, tags: :membrane_element_aac
+
+  # AAC Constants
+  @aac_frame_size 1024
```

Comment:
@bblaszkow06 One more thing I've noticed.
It seems that 1024 only works for AAC-LC profile and all others (HE profiles) use 2048.
I couldn't find any reliable info about that, but this seemed consistent when I tried different profiles.

Moreover, MPEG-4 AAC Low-Delay (23) and MPEG-4 AAC Enhanced Low-Delay (39) return "Transport library initialization error" from the native (fdk-aac) encoder. If we really need to support them I can investigate this a little bit more, but maybe we would be fine dropping those profiles for now (at least for v1).